contracts/governance/Governor.sol

```diff
@@ -207,14 +203,16 @@ abstract contract Governor is Context, ERC165, EIP712, IGovernor, IERC721Receive
      * @dev See {IGovernor-proposalSnapshot}.
      */
     function proposalSnapshot(uint256 proposalId) public view virtual override returns (uint256) {
-        return _proposals[proposalId].voteStart;
+        ProposalCore memory proposal = _proposals[proposalId];
+        return proposal.voteStart;
```

Comment:
Why would this be better? It implies an additional memory allocation. I think the previous code would definitely be more efficient.

Reply:
Here it doesn't bring anything. I did it for consistency with the other functions, but we ca revert to direct read.

Reply:
```suggestion
        return _proposals[proposalId].voteStart;
```